applications/tari_merge_mining_proxy/README.md

```diff
@@ -21,25 +25,17 @@ Follow build instructions for XMRig here:
 Then create a configuration for XMRig using the wizard here:
 `https://xmrig.com/wizard`
 
-When using the wizard you need to use `Add daemon` and the following public stagenet wallet address can be used:
-`55LTR8KniP4LQGJSPtbYDacR7dz8RBFnsfAKMaMuwUNYX6aQbBcovzDPyrQF9KXF9tVU6Xk3K8no1BywnJX6GvZX8yJsXvt`
-
-Take note of the `Host` and `Port` that was specified in the configuration.
+When using the wizard you need to use `Add daemon`. The following public stagenet wallet address can be used (or you can use your own stagenet wallet address):
+`55LTR8KniP4LQGJSPtbYDacR7dz8RBFnsfAKMaMuwUNYX6aQbBcovzDPyrQF9KXF9tVU6Xk3K8no1BywnJX6GvZX8yJsXvt`. The `Host` and `Port` fields can be filled in according to the 
+value of `proxy_host_address` that was previously noted down.
 
-You will put the resulting `config.json` file together with the built binaries for XMRig.
+Once you have finished the wizard, you will put the resulting `config.json` file together with the built binaries for XMRig.
 
-Start XMRig from the terminal using `./xmrig`
+Start XMRig from the terminal using `./xmrig`.
 
 ### Tari Merge Mining Proxy
```

Comment:
Shouldn't this be started before xmrig?

Reply:
Can be started before or after, what matters is that everything the proxy communicates to is started before it.

Have swapped the sections around.